plaidml/op/op.h

```diff
@@ -527,6 +534,12 @@ class mvn {
   std::string layout_;
 };
 
+// NMS
+// params
+// @nms_style: There are currently 3 styles of nms. OpenVINO, Caffe and MxNet. OV style is the most common one and
```

Comment:
I think it may be better to move the explaidnation of  nms_style to the nms_style function. If you add explaidnation here, it is better to add for all parameters.